python/ray/tune/examples/cifar10_pytorch.py

```diff
@@ -178,7 +180,12 @@ def main(num_samples=10, max_num_epochs=10, gpus_per_trial=2):
         "l1": tune.sample_from(lambda _: 2 ** np.random.randint(2, 9)),
         "l2": tune.sample_from(lambda _: 2 ** np.random.randint(2, 9)),
         "lr": tune.loguniform(1e-4, 1e-1),
-        "batch_size": tune.choice([2, 4, 8, 16])
+        "batch_size": tune.choice([2, 4, 8, 16]),
+        "wandb": {
+            "project": "CIFAR10",
+            "api_key": "1f06d7f72cff2b61773efd8d1d151e71b3c39885",
+            "group_by": ["batch_size"]
```

Comment:
we don't need this right?

Reply:
Yeah I think it makes configuration easier if we have less options there, so I will remove this later